tools/codeformat.py

```diff
@@ -161,6 +166,16 @@ def main():
     # Expand the globs passed on the command line, or use the default globs above.
     files = []
     if args.files:
+        if args.f:
+            all_files = list_files(PATHS, EXCLUSIONS, TOP)
+            filtered = [
+                f for f in args.files if os.path.normpath(os.path.join(TOP, f)) in all_files
+            ]
+            for r in args.files:
+                if args.v and r not in filtered:
+                    print("Not checking {}".format(r))
```

Comment:
Not sure it's worth making this verbose, the rest of the tool is also silent and excludes files without saying so. But maybe there's a reason you did this?

Reply:
If you pass `-v` then uncrustify and black will each print a line for each file they check, even though nothing is printed by the Python script.

If this line is printed, it means the user passed a specific file name on the command line but the tool decided to ignore that file. IMO it's good to have a way to show the user that this is happening, in the spirit of "principle of least surprise".

Reply:
Sorry again, I missed the `if arg.v` part :)

Reply:
No worries, thanks for looking over the PR! :+1: